src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -13,30 +13,81 @@ public class CartPage {
     private final By productName_h4 = By.xpath("//td[@class='cart_description']//h4");
     private final By viewCart_a = By.xpath("//div[contains(@class,'confirm')]//a[@href='/view_cart']");
 
+    private String url = System.getProperty("baseUrl") + "/view_cart";
+
+    private final By firstProductName = By.xpath("//a[normalize-space()='Blue Top']");
+
+    private final By secondProductName = By.xpath("//a[normalize-space()='Men Tshirt']");
+
+    private final By firstProductPrice = By.xpath("//td[@class='cart_price']//p[contains(text(),'Rs. 500')]");
+
+    private final By secondProductPrice = By.xpath("//td[@class='cart_price']//p[contains(text(),'Rs. 400')]");
+
+    private final By firstProductQuantity = By.xpath("//tr[@id='product-1']//button[@class='disabled'][normalize-space()='1']");
+
+    private final By secondProductQuantity = By.xpath("//tr[@id='product-2']//button[@class='disabled'][normalize-space()='1']");
+
+    private final By firstProductTotal = By.xpath("//p[@class='cart_total_price'][normalize-space()='Rs. 500']");
+
+    private final By secondProductTotal = By.xpath("//p[@class='cart_total_price'][normalize-space()='Rs. 400']");
+
     // Constructor
-    public CartPage(SHAFT.GUI.WebDriver driver){
+    public CartPage(SHAFT.GUI.WebDriver driver) {
         this.driver = driver;
     }
 
     //////////////////// Actions \\\\\\\\\\\\\\\\\\\\
 
     //Clicking using JS as fix for pipeline failure on safari (Click isn't happening even with ClickUsingJS Flag)
     @Step("Open Cart Page")
-    public CartPage openCart(){
+    public CartPage openCart() {
         driver.element().clickUsingJavascript(viewCart_a);
         return this;
     }
 
     //////////////////// Validations \\\\\\\\\\\\\\\\\\\\
     @Step("Verify Cart Page is Loaded")
-    public CartPage verifyCartPageIsLoaded(){
+    public CartPage verifyCartPageIsLoaded() {
         driver.verifyThat().element(proceedToCheckout_btn).isVisible().perform();
         return this;
     }
 
     @Step("Validate on Product Added To Cart Page")
-    public CartPage verifyProductAddedToCart(String addedProductName){
+    public CartPage verifyProductAddedToCart(String addedProductName) {
         driver.assertThat().element(productName_h4).text().isEqualTo(addedProductName).perform();
         return this;
     }
+
+    public CartPage navigate() {
+        driver.browser().navigateToURL(url);
+        return this;
+    }
```

Comment:
navigation method is not a validation method, please move it to the top on the actions section.